index.html

```diff
@@ -3246,6 +3246,55 @@ <h3>Authorization</h3>
 on top of this specification.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Reserved Extension Points</h3>
+
+        <p>
+This specification reserves a number of <a>properties</a> serving as possible extension points. While
+some implementers signaled interest in these, their inclusion in this specification
+was considered to be premature; these extension points might be more
+formally defined in future versions of this specification. While these extension
+points are reserved in the base context, their usage is undefined by this
+specification and implementers are cautioned that usage of these properties is
+considered experimental.
+        </p>
+
+        <p class="issue" title="Reserved properties under debate">
+The following extension point properties are under consideration for being
+marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
+`evidence`, `render`, and `confirmationMethod`.
```

Comment:
```suggestion
`evidence`, `render`, `refreshService` and `confirmationMethod`.
```
refreshService should also be included

Reply:
Agreed

Reply:
Does this mean that `credentialSchema` and `credentialStatus` are the only extension points we intend to both reserve and define?

Reply:
`refreshService` already has at least two independent demonstrable implementations (that are locked up behind NDAs, unfortunately). `credentialSchema` and `credentialStatus` don't yet, but both are WG work items.

So, perhaps the rule here is: 

1. If it's not a WG work item, or it doesn't already have two independent demonstrable interoperable implementations, it goes on the "reserved extension points" list. OR
2. If it's not a WG work item, it goes on the "reserved extension points" list.

Reply:
> Does this mean that `credentialSchema` and `credentialStatus` are the only extension points we intend to both reserve and define?

Yes, seems so.

Reply:
@msporny  credentialSchema already has several multiple independent implementations since this was a required property of the JFF plugfest. Multiple suppliers issued VCs with the credentialSchema property in it, and multiple suppliers provided wallets that validated the credentialSchema property. These are not locked up in NDAs and the interworking table of all the suppliers was made public available here https://docs.google.com/spreadsheets/d/1VNsQT_S_dQ1JnStvIJOGo5uuLtjI8m0z1YGS9q55YYk/edit#gid=152952696